ui/dialog.py

```diff
@@ -139,31 +139,36 @@ def refresh(self):
         self.o.display_data(self.el.message, self.displayed_label)
         self.o.cursor()
 
-
 class GraphicalView(TextView):
 
     def get_image(self):
         c = Canvas(self.o)
 
         #Drawing text
-        second_line_position = 10
-        ctc = c.get_centered_text_bounds(self.el.message)
-        c.text(self.el.message, (ctc.left, 0))
-        c.text(self.displayed_label, (2, second_line_position))
+        choice_position = 10
+        chunk_y = 0
+        rendered_message = []
+        while self.el.message:
+                rendered_message.append(self.el.message[:self.el.o.cols])
+                self.el.message = self.el.message[self.el.o.cols:]
+                choice_position += 10
```

Comment:
I suggest you make this hardcoded `10` into a separate GraphicalView attribute

Reply:
okay, will do

Reply:
Or would o.char_height + 2 do the job?